sympy/utilities/lambdify.py

```diff
@@ -151,10 +149,20 @@ def _import(module, reload="False"):
     for sympyname, translation in translations.items():
         namespace[sympyname] = namespace[translation]
 
+    # For computing the modulus of a sympy expression we use the builtin abs
+    # function, instead of the previously used fabs function for all
+    # translation modules. This is because the fabs function in the math
+    # module does not accept complex valued arguments. (see issue 9474). The
+    # only exception, where we don't use the builtin abs function is the
+    # mpmath translation module, because mpmath.fabs returns mpf objects in
+    # contrast to abs().
```

Comment:
Here we use mpmath.fabs because it returns mpf objects even for non-mpmath arguments. (This is not possible by overriding __abs__ in mpmath).  To be honest, I don't know if we need it. Take a look at https://mpmath.googlecode.com/svn/tags/0.11/doc/build/functions/numberparts.html